.gitignore

```diff
@@ -0,0 +1,9 @@
+.directory
+__pycache__
+pull_request_Information.md
+Datasets/State_of_OA/Not_Currently_Used
+Datasets/State_of_OA/state-of-oa-dois_CLEAN.tsv
+library_coverage_xml_and_fulltext_indicators.db*
```

Comment:
This ignore should stay... eventually we'll switch it to only ignore the uncompressed db and a compressed db will be for archiving.

Reply:
I'm leaving a note here that Write-Ahead Logging (WAL) for SQLite is enabled in the code, so when the code is run, two additional files are generated, `library_coverage_xml_and_fulltext_indicators.db-shm` and `library_coverage_xml_and_fulltext_indicators.db-wal`.